### DISCUSSION TITLE
Where did my entries go?

### DISCUSSION THREAD

**Discussion by plolock** [kimai/kimai#5195] [Questions & Answers]
I was in the Export view, and had filtered on a project for last month.
It was a list of maybe 12 activites or logs, and they all had a button on the far right that said "open".

For some reason, dont ask me why (I guess its because I'm trying to figure out how the app works) I clicked them all - and now they wont show in the export view. 

What exactly have I dont and what happened to the things that was in the list that I in fact wanted to be there, but now they are not?

Thanks!

**Comment by kevinpapst** [comment]
You Exported them, so they are filtered out.
Check the search form and switch "Exported" from "No" to "Yes" or "All"
<img width="614" alt="Bildschirmfoto 2024-12-03 um 20 50 24" src="https://github.com/user-attachments/assets/e20e82bb-46b0-46ef-b419-ca516b9088a2">

**Reply by plolock** [reply]
Thanks a bunch!
